hooks/pre-commit

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+from __future__ import print_function
+from future import standard_library
+standard_library.install_aliases()
+
+import os
+import urllib.request
+import urllib.parse
+import urllib.error
```

Comment:
I kept the urllib.request in because our initialization of version_file requires it, but otherwise got rid of the other imports!